### PR TITLE
More Solidity IDE links

### PR DESCRIPTION
**Pull request by winsvega**

### CODE REVIEW

**Comment by chriseth** [comment]
Please rebase instead of merging.

File: docs/index.rst [hunk 1]
```diff
@@ -28,6 +26,19 @@ Useful links
 
 * `Gitter Chat <https://gitter.im/ethereum/solidity/>`_
 
+Available IDE
```
Comment (chriseth) [inline docs/index.rst]: `IDEs`?

File: docs/index.rst [hunk 2]
```diff
@@ -28,6 +26,19 @@ Useful links
...
+-------------
+* `Ethereum Studio <https://live.ether.camp/>`_
+
+* `Browser-Based Compiler <https://chriseth.github.io/browser-solidity/>`_
+
+* `VS Extention <https://visualstudiogallery.msdn.microsoft.com/96221853-33c4-4531-bdd5-d2ea5acc4799/>`_
```
Comment (chriseth) [inline docs/index.rst]: Please use the term `Visual Studio`

File: docs/index.rst [hunk 3]
```diff
@@ -28,6 +26,19 @@ Useful links
...
+
+* `Mix IDE <https://github.com/ethereum/mix/>`_
+
+* `MeteorJS development environment DApp <https://github.com/SilentCicero/meteor-dapp-cosmo/>`_
+
```
Comment (chriseth) [inline docs/index.rst:48]: There should also be extensions for atom, syntax hilighting for sublime and of course the Visual Studio Code extension.

**Comment by chriseth** [comment]
Visual studio code is still missing.

**Comment by winsvega** [comment]
you mean Visual Studio Extention ? it is there

File: docs/index.rst [hunk 4]
```diff
@@ -28,6 +26,25 @@ Useful links
 
 * `Gitter Chat <https://gitter.im/ethereum/solidity/>`_
 
+Available IDEs
+-------------
```
Comment (chriseth) [inline docs/index.rst]: You need another `-` here.

File: docs/index.rst [hunk 5]
```diff
@@ -28,6 +26,25 @@ Useful links
...
+* `Ethereum Studio <https://live.ether.camp/>`_
+
+* `Browser-Based Compiler <https://chriseth.github.io/browser-solidity/>`_
+
+* `Visual Studio Extention <https://visualstudiogallery.msdn.microsoft.com/96221853-33c4-4531-bdd5-d2ea5acc4799/>`_
```
Comment (chriseth) [inline docs/index.rst]: `Extension` and perhaps "Microsoft Visual Studio`?

**Comment by chriseth** [comment]
Please add more information, i.e. whether the IDE can simulate function calls and whether it can compile and show errors or just do syntax hilighting.

Also cosmo seems to be inactive, so I think we can remove it.

File: docs/index.rst [hunk 6]
```diff
@@ -28,6 +26,25 @@ Useful links
 
 * `Gitter Chat <https://gitter.im/ethereum/solidity/>`_
 
+Available IDEs
+-------------
+* `Ethereum Studio <https://live.ether.camp/>`_
+
+* `Browser-Based Compiler <https://chriseth.github.io/browser-solidity/>`_
```
Comment (axic) [inline docs/index.rst:31]: Shouldn't this be the first as it is related to the Solidity project? Maybe a better (more meaningful) name could be used, e.g. _In-browser Solidity compiler and simulator_?

Alternatively using alphabetical order?

Comment (VoR0220) [inline docs/index.rst:31]: agreed. We should put the browser-solidity first...frankly it works best. Aside from Mix there's no better place to develop your solidity in. :P Foundation privileges.

Comment (winsvega) [inline docs/index.rst:31]: What about Ethereum Studio?

File: docs/index.rst [hunk 7]
```diff
@@ -28,6 +26,30 @@ Useful links
 
 * `Gitter Chat <https://gitter.im/ethereum/solidity/>`_
 
+Available Solidity Integrations
+-------------
```
Comment (chriseth) [inline docs/index.rst]: The `----` is too short.

File: docs/index.rst [hunk 8]
```diff
@@ -28,6 +26,30 @@ Useful links
...
+* `Browser-Based Compiler <https://chriseth.github.io/browser-solidity/>`_
+	WEB compiler for solidity language which could also simulate function calls.
```
Comment (chriseth) [inline docs/index.rst]: `Browser-based IDE with integrated compiler and Solidity runtime environment without server-side components`

File: docs/index.rst [hunk 9]
```diff
@@ -28,6 +26,30 @@ Useful links
 
 * `Gitter Chat <https://gitter.im/ethereum/solidity/>`_
 
+Available Solidity Integrations
+-------------
+* `Browser-Based Compiler <https://chriseth.github.io/browser-solidity/>`_
+	WEB compiler for solidity language which could also simulate function calls.	
+
+* `Ethereum Studio <https://live.ether.camp/>`_
+	A fully-functional well-designed WEB IDE for Ethereum Solidity developers.
```
Comment (chriseth) [inline docs/index.rst]: `Specialized web IDE that also provides shell access to a complete Ethereum environment.`

File: docs/index.rst [hunk 10]
```diff
@@ -28,6 +26,30 @@ Useful links
...
+
+* `Visual Studio Extention <https://visualstudiogallery.msdn.microsoft.com/96221853-33c4-4531-bdd5-d2ea5acc4799/>`_
+	Consensys integration of Solidity into Microsoft Visual Studio 2015
```
Comment (chriseth) [inline docs/index.rst]: I think it is not too relevant who made this.

`Solidity plugin for Microsoft Visual Studio that includes the Solidity compiler.`

File: docs/index.rst [hunk 11]
```diff
@@ -28,6 +26,30 @@ Useful links
...
+
+* `Mix IDE <https://github.com/ethereum/mix/>`_
+	A developer's IDE for designing, debugging and testing solidity smart contracts.
```
Comment (chriseth) [inline docs/index.rst]: `Qt based IDE for ...`

File: docs/index.rst [hunk 12]
```diff
@@ -28,6 +26,30 @@ Useful links
...
 
+Available Solidity Integrations
+-------------
+* `Browser-Based Compiler <https://chriseth.github.io/browser-solidity/>`_
+	WEB compiler for solidity language which could also simulate function calls.	
+
+* `Ethereum Studio <https://live.ether.camp/>`_
+	A fully-functional well-designed WEB IDE for Ethereum Solidity developers.
+
+* `Visual Studio Extention <https://visualstudiogallery.msdn.microsoft.com/96221853-33c4-4531-bdd5-d2ea5acc4799/>`_
```
Comment (chriseth) [inline docs/index.rst]: `Extension`

File: docs/index.rst [hunk 13]
```diff
@@ -28,6 +26,30 @@ Useful links
...
+	A developer's IDE for designing, debugging and testing solidity smart contracts.
+
+* `Package for SublimeText — Solidity language syntax <https://packagecontrol.io/packages/Ethereum/>`_
+	Solidity syntax highlight for SublimeText editor.
+
+* `Atom Solidity package <https://github.com/gmtcreators/atom-solidity/>`_
+	Analog of solidity browser-based compiler for atom editor.
+
+* `Visual Studio Code extension <http://juan.blanco.ws/solidity-contracts-in-visual-studio-code/>`_
+	Solidity syntax highlight and debugging for Visual Studio Code.
```
Comment (chriseth) [inline docs/index.rst]: `Solidity plugin for Microsoft Visual Studio Code that includes syntax highlighting and the Solidity compiler.`

File: docs/index.rst [hunk 14]
```diff
@@ -28,6 +26,30 @@ Useful links
...
+	Consensys integration of Solidity into Microsoft Visual Studio 2015
+
+* `Mix IDE <https://github.com/ethereum/mix/>`_
+	A developer's IDE for designing, debugging and testing solidity smart contracts.
+
+* `Package for SublimeText — Solidity language syntax <https://packagecontrol.io/packages/Ethereum/>`_
+	Solidity syntax highlight for SublimeText editor.
+
+* `Atom Solidity package <https://github.com/gmtcreators/atom-solidity/>`_
+	Analog of solidity browser-based compiler for atom editor.
```
Comment (chriseth) [inline docs/index.rst]: `Plugin for the atom editor that features syntax highlighting, compilation and a runtime environment (requires backend node).`

File: docs/index.rst (hunk 8 above)
Comment (axic) [inline docs/index.rst]: Shouldn't this also point to the new URL?

**Pull request merged**